internal/provider/auth_gcp.go

```diff
@@ -123,7 +127,7 @@ func (l *AuthLoginGCP) getJWT(ctx context.Context) (string, error) {
 
 	if v, ok := l.params[consts.FieldCredentials]; ok && v.(string) != "" {
 		// get the token from IAM
-		creds, err := getGCPOauthCredentials(ctx, v.(string))
+		creds, err := getGCPOauthCredentials(ctx, v.(string), defaultGCPScope)
```

Comment:
Getting the scopes from the configured params might make better sense here, since it would allow the user to set whatever they'd like. See the related comment above.